CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessPane.java

```diff
@@ -180,7 +180,7 @@ public DataAccessPane(DataAccessViewTopComponent topComponent) {
                 .addListener((oldValue, newValue) -> update());
 
         final Button helpButton = new Button("", new ImageView(UserInterfaceIconProvider.HELP.buildImage(16, ConstellationColor.BLUEBERRY.getJavaColor())));
-        helpButton.paddingProperty().set(HELP_INSETS);
+        helpButton.paddingProperty().set(new Insets(0, 8, 0, 64));
```

Comment:
Set `HELP_INSETS` to this new value and then use the constant

Reply:
@aldebaran30701 made that change to only fix the options help icon. Changing the constant to be the new value will modify all of the padding to all of the icons which pushes them into an awkward position. Would it be a good idea to create a separate constant and store the value in this just for the options help icon?

Reply:
Ah ok. When I did my review I only saw the constant only being applied here so I must've missed that.
For what it's worth, I probably wouldn't bother creating a constant simply just for this unless you reckon this value will be used else where.